kuma/static/styles/includes-skinny/_mixins.scss

```diff
@@ -332,6 +332,27 @@ Spacing
 Bidi / l10n
 ====================================================================== */
 
+@mixin bidi($list) {
```

Comment:
Would be great to add some documentation here with a sample of a use case and the code that is produced.